tests/test_cases/test_multi_trig_multi_coro/test_multi_trig_multi_coro.py

```diff
@@ -0,0 +1,58 @@
+import cocotb
+from cocotb.log import SimLog
+from cocotb.triggers import Timer, RisingEdge, FallingEdge
+from cocotb.result import TestFailure, TestSuccess
+
+import sys
+
+@cocotb.coroutine
+def clock_gen(signal, num):
+    for x in range(num):
+        signal <= 0
+        yield Timer(500)
+        signal <= 1
+        yield Timer(500)
+
+class DualMonitor:
+    def __init__(self, signal):
+        self.log = SimLog("cocotb.%s" % (signal._name))
```

Comment:
nit: `% (x)` doesn't do what you think it does, use `% (x,)` or `.format(x)` instead.